### PR TITLE
Disable <meta http-equiv=set-cookie>

### PR DESCRIPTION
**Pull request by g4jc**
Sets a new preference: `dom.meta-set-cookie.enabled` to `false`, by default. Fixing #1102.

### CODE REVIEW

**Comment by mattatobin** [comment]
As far as prefs like this is conserned the only pref naming convention is from dom.meta-viewport.enabled..

Can we have the pref named dom.meta-setCookie.enabled instead?

What is your opinion, @wolfbeast?

**Comment by wolfbeast** [comment]
I agree that the current pref name isn't really good. Let's make it `dom.meta-set-cookie.enabled` as a logical name for enabling setting of cookies via meta.

As for false by default, I think we're good on that.

**Comment by mattatobin** [comment]
I prefer my pref name better but I am fine with Moonchild's. Anything is better than what I assume Mozilla used.

**Comment by g4jc** [comment]
Pref name changed as requested. Not sure why github closed the PR?

**Comment by g4jc** [comment]
Confirming bug in [#361 Github](https://github.com/isaacs/github/issues/361). TIL. Opening a new PR.

**Comment by wolfbeast** [comment]
Force-pushing branches overwriting history makes things really hard to follow and requires pretty much a re-review of the entire PR after it. I'd prefer if you didn't do so in the future and just add changes as commits to the existing branch the PR is from -- they will automatically be picked up as part of the PR.

**Comment by wolfbeast** [comment]
This resolves #1102
